_posts/2019-02-22-Caring-For-The-Publisher.md

```diff
@@ -11,51 +11,52 @@ header-img: "images/Caring-For-The-Publisher/header-bg.jpg"
 As an introductory article to the frontend section of the blog, we would like to give a quick overview of some of the best practices that our creatives go through before being delivered to various publishers that come in many forms and sizes.
 
 The speed of the publisher web pages can be often hurt by 3rd party content, the publishers having somewhat no control over what the content loads on its end, how big the files are or how that impacts the end-user experience.
-Here at Adello, not only that we care how our creatives impact the publisher's overall page performance, we are also very aware that what helps the publisher, helps us, and two things would be load speed and seamless integration.
+
+Here at Adello, we do not only care how our creatives impact the publisher's overall page performance. We are also very aware that what helps the publisher, helps us. Two main points would be load speed and seamless integration.
 So to ensure that our creatives load seamlessly in their environment, here are a few rules of thumb we follow that are mainly categorized into two aspects, best practices and improvements.
 
 ### Vanilla Code
-It is tempting to jump over using online libraries to get things done quicker and easier, such as jQuery, Bootstrap, GreenSock to name a few, but if they are not heavily used, such as in a full-featured website, then they add too much overhead and impact load and browsing performance.
-Taking jQuery as example, nowadays, JavaScript is better supported in most browsers and we no longer have to cover so much cross-browser compatibility solutions. Already, a wide number of jQuery methods are supported natively by JavaScript and I don't need to say this, but native is better, and faster!
+It is tempting to jump over using online libraries to get things done quicker and easier, such as jQuery, Bootstrap, GreenSock to name a few. However, if they are not heavily used, such as in a full-featured website, then they add too much overhead and impact load and browsing performance.<br>
+Taking [jQuery](https://jquery.com/) as example, nowadays, JavaScript is better supported in most browsers and we no longer have to cover so much cross-browser compatibility solutions. Already, a wide number of jQuery methods are supported natively by JavaScript and I don't need to say this, but native is better, and faster!
```

Comment:
I would also add  bit about jQuery. Given that it is widely used, if we try to use it in our ad, it could conflict with the version on the publisher as well.

Reply:
While that is not the reason we are not using jQuery, it is a good thing to note indeed, so I have added it in the form of an advice. Let me know how it looks